pkg/reconcilermanager/controllers/reconciler_base.go

```diff
@@ -426,7 +426,7 @@ func (r *reconcilerBase) deployment(ctx context.Context, dRef client.ObjectKey)
 	return deployObj, nil
 }
 
-func mutateContainerResource(ctx context.Context, c *corev1.Container, override *v1beta1.OverrideSpec, reconcilerType string) {
```

Comment:
So these variables were unused? I'm surprised the linter didn't complain

Reply:
Yes, those are unused. Reporting unused function parameters is done via the `unparam` option, which is disabled by default.

I tried to turn it on locally, however, the `always receives` check is annoying: https://github.com/mvdan/unparam/issues/31. Once we're able to disable the `always receives` check, we can turn on the `unparam` check.